transaction/examples/call_method.rtm

```diff
@@ -0,0 +1,38 @@
+# Inovke scrypto method (both global & and)
+CALL_METHOD ComponentAddress("component_sim1qgvyxt5rrjhwctw7krgmgkrhv82zuamcqkq75tkkrwgs00m736") "free_xrd";
+CALL_METHOD Component("000000000000000000000000000000000000000000000000000000000000000005000000") "free_xrd";
+
+# Invoke native method (ref)
+TAKE_FROM_WORKTOP ResourceAddress("resource_sim1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzqu57yag") Bucket("xrd");
+CREATE_PROOF_FROM_AUTH_ZONE ResourceAddress("resource_sim1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzqu57yag") Proof("proof");
+CALL_NATIVE_METHOD &Bucket("xrd") "get_resource_address";
```

Comment:
I think seems wrong to include `&` in these. Compiler should be smart enough to know.

Reply:
The idea was to give some visual representation of passing by reference, in distinction from passing by ownership. Since we're removing support of passing by ownership, I don't mind removing the `&` by that time.

Also, I didn't want to introduce too much compiler inference about nagive funciton/method ABI, i.e. the compiler/decompiler should continue to work even if we've added/deleted native methods.